test/unit/metadata.js

```diff
@@ -772,4 +774,67 @@ describe('metadata', function () {
       should(upToDate(doc).errors).be.undefined()
     })
   })
+
+  describe('generated a doc with conflict', () => {
```

Comment:
Maybe use the function name as the description, same as in other unit tests?

Reply:
Why add more coupling? What if the function name changes?

Reply:
It allows guessing the test filter from the function name without even looking at the test description:

```bash
yarn test:unit -g createConflictingDoc
# Vs
yarn test:unit -g 'generated a doc with conflict'
```

Reply:
(if the function name changes, we'll have to change it in each test anyway)

Reply:
If the function name changes, my editor's tools that help me renaming things will rename things, but not the comments.

Reply:
Doesn't matter so much. Please forget about my comment :)